install_ffmpeg.sh

```diff
@@ -137,8 +137,8 @@ if [ ! -e "$ROOT/ffmpeg/libavcodec/libavcodec.a" ]; then
     --enable-muxer=mpegts,hls,segment,mp4,null --enable-demuxer=flv,mpegts,mp4,mov \
     --enable-bsf=h264_mp4toannexb,aac_adtstoasc,h264_metadata,h264_redundant_pps,extract_extradata \
     --enable-parser=aac,aac_latm,h264 \
-    --enable-filter=abuffer,buffer,abuffersink,buffersink,afifo,fifo,aformat \
-    --enable-filter=aresample,asetnsamples,fps,scale,lvpdnn \
+    --enable-filter=abuffer,buffer,abuffersink,buffersink,afifo,fifo,aformat,format \
```

Comment:
I might've missed this in the LPMS PR review - what is the `format` filter used for?

Reply:
The `format` filter is needed after `hwdownload` filter to signal that we want to convert CUDA frames to `NV12` pixel format 
https://ffmpeg.org/ffmpeg-filters.html#hwdownload-1 used [here in lpms](https://github.com/livepeer/lpms/commit/e85ff00f8df274bacc9bca2a154493800979b275#diff-4ab68a4de67768f1acb5be217d4ec78bab27b99381025beedf82c11a5fa9c7aeR310)

Reply:
Ah right! Thanks for the explanation.